### DISCUSSION TITLE
AssertionError: ASCII operator to Unicode tables are missing from C:/data/mingw64/lib/python3.11/site-packages/mathics/data/op-tables.json

### DISCUSSION THREAD

**Discussion by Kreijstal** [Mathics3/mathics-core#1033] [Q&A] [answered]
I installed mathics-core using `pip install .`

Googling did not get any meaningful results, where can I get this file?

**Comment by rocky** [comment]
When installing from source, use `pip install -e .`  

This is mentioned in https://mathics-development-guide.readthedocs.io/en/latest/installing.html#id10

It is possible though that the instructions might need to be updated. You might have to run `make` to build these tables.

**Comment by Kreijstal** [comment, marked as answer]
Painless installation on windows using msys2:
git clone, go to dir
```
pacman -S --needed pactoys
pacboy -S --needed python-llvmlite:p \
python-mpmath:p \
python-numpy:p \
python-pillow:p \
python-dateutil:p \
python-requests:p \
python-setuptools:p \
python-sympy:p \
python-pexpect:p \
python-pytest:p \
python-click:p \
python-ptyprocess:p \
python-iniconfig:p \
python-packaging:p \
python-pluggy:p \
python-colorama:p \
python-six:p \
python-idna:p \
python-urllib3:p \
python-yaml:p
pip install .
make
```

**Reply by rocky** [reply]
Great,  Good to hear!  Would you consider revising instructions  - perhaps adding a section for msys2 in the mathics-development-guide  - https://github.com/Mathics3/mathics-development-guide/blob/master/docs/installing.rst ? 

Thanks in advance.
